go_backend/setup.sql

```diff
@@ -0,0 +1,36 @@
+CREATE database IF NOT EXISTS foodApp
```

Comment:
We need to drop the whole DB when the schema changes. Use DROP DATABASE IF EXISTS then create a new one